paper/appendix.tex

```diff
@@ -0,0 +1,177 @@
+\appendix{From A Posterior with Gaussian Distributions to an least-squares Objective-Function}
```

Comment:
Regarding the `appendix ??` problem: Currently MyST does not recognize `\appendix`. This means these appendix headings are just text, not numbered sections.

For now, you can just swap `\appendix` with `\section` and the cross-reference resolves correctly. For this paper, it seems like this change will be fine; the appendices are all on the "appendix" page, and all the cross-references say "appendix #"

```suggestion
\section{From A Posterior with Gaussian Distributions to an least-squares Objective-Function}
```